extension/src/setup/index.ts

```diff
@@ -280,13 +289,16 @@ export class Setup
   private setProjectAvailability() {
     const available = this.hasRoots()
     setContextValue('dvc.project.available', available)
+    if (available && this.dotFolderWatcher && !this.dotFolderWatcher.disposed) {
```

Comment:
[F] If there is a project available and the watcher hasn't been disposed yet then get rid of it